Source/WebCore/rendering/RenderObject.h

```diff
@@ -1210,6 +1212,14 @@ inline void Node::setRenderer(RenderObject* renderer)
         notifyInspectorOfRendererChange();
 }
 
+inline void RenderObject::notifyInspectorOfPropertyChange()
+{
+    if (UNLIKELY(InspectorInstrumentationPublic::hasFrontends())) {
+        if (auto* node = this->node())
+            node->notifyInspectorOfRendererPropertyChange();
```

Comment:
ditto (Source/WebCore/page/FrameView.cpp:3002) by having the callsites just `InspectorInstrumentation::rendererPropertyChanged(...)`